### ISSUE TITLE
No title is shown in edit mode if note's title starts with a double quote ( " )

### ISSUE THREAD

**Issue by Atomk** [Laverna/laverna#607]
1) I have created a new note, the title starts with a double quote.
2) I select it from the note list, press "edit", the edit screen opens up but shows an empty title, meaning the only solution is to write a new title that starts with a letter since the editor does not let you save a note without a title.

**Comment by JerJohn15** [comment]
@Atomk @wwebfor   This issue appears to be a duplicate to #540. According to the discussions, there was a Pull Request by @janis-kra to fix this issue but the bug still exists (I've verified it as well). I think that issue needs to be reopened.

**Comment by claell** [comment]
I don't think it is a duplicate of #540 as the title is not empty when attempting to save.

**Comment by JerJohn15** [comment]
@claell Your right, my mistake. So the test case for this should be:

1. When verifying a title name include a double quote (" ").

2. If one exists, save the string  in the variable that contains a title name. 

3. When editing a note, read the title name from the variable that saved it and display it in the title field.

I want to attempt to resolve this issue.

**Comment by claell** [comment]
I don't know what is causing the issue, so maybe there is an easier fix.
